internal/MozziGuts_impl_RP2040.hpp

```diff
@@ -217,9 +248,10 @@ static void startAudio() {
 #if MOZZI_IS(MOZZI_AUDIO_MODE, MOZZI_OUTPUT_PWM)
   // calling analogWrite for the first time will try to init the pwm frequency and range on all pins. We don't want that happening after we've set up our own,
```

Comment:
I suppose this comment block is essentially obsolete, then?

Reply:
Indeeed